docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
 
-The Product Delivery Model focuses on the customer experience, not on the technical artifacts you're delivering, and focuses on making it possible to deliver a first-class, fully configured customer experience – no matter how complex the product might be. When building a product, there can be detailed technical considerations for whether a specific component is best distributed within a package, as additional unpackaged metadata, or as sophisticated setup automation that runs before or after a package is installed. And, of course, a product may include not one, but several packages, which must be installed in sequence. CumulusCI makes it possible to deliver a cohesive product that makes use of these different techniques.
+Don't worry, we'll make this as painless as possible.
 
-CumulusCI provides the core framework of operations for automating the Product Delivery Model throughout the development lifecycle, and a command line interface to those operations. Once configured, the same automation can also be used by other applications in the CumulusCI Suite, such as MetaCI for automated testing and MetaDeploy for delivering products to customers.
-
-The Product Delivery Model aims to represent a holistic view of delivery of a product instead of simply releasing a package. Here's a real-world example drawn from Salesforce.org's product portfolio.
-
-The Nonprofit Success Pack and the Product Delivery Model
-++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-
-The Nonprofit Success Pack, or NPSP, is one of Salesforce.org's flagship products. The NPSP consists of six managed packages, with complex dependency relationships between them, alongside unpackaged metadata and setup automation that helps create a usable NPSP org.
+Packages
+--------
 
-Salesforce.org delivers NPSP using CumulusCI and the Product Delivery Model because those tools are what make it possible for us to ship this large, complex, heterogeneous application efficiently to tens of thousands of customers — without a lengthy setup guide to be completed by the end user. CumulusCI automation seamlessly
+If you've come this far in your search of customization, we're guessing you're familiar with packages as well as their role in customizing a Salesforce org. However, because most everything that is built in CumulusCI is deployed via packages, it's essential to clarify once more:
 
-* deploys NPSP's five dependent managed packages in the right sequence;
-* delivers unpackaged Record Types for the Account and Opportunity objects;
-* installs NPSP;
-* and performs final configuration to make the customer's experience better, like setting up Global Actions and delivering translations.
+A **package** is a container for something as small as an individual component or as large as a set of related apps. After creating a package, you can distribute it to other Salesforce users and organizations, including those outside your company.
```

Comment:
The typical Salesforce standard for indicating emphasis and indicating a newly introduced term is italics. We do sometimes use bold for emphasis, though not with any consistent pattern that I can detect. Flagging it for now as a formatting question for this whole doc set. I see David Reed also called out a formatting question about formatting code. We should discuss.

Reply:
Yeah, wasn't sure if bold or italics was the way to go. I figured bold would stand out better because these are VIP key concepts.

Reply:
Let's see if @jvarese has an opinion.

Reply:
@justinvholt - as written, the above is definitely an italics situation (per house style). We use italics in the flow of a sentence. Another way to extra emphasize something is with a definition list, which bolds the first word/definition automatically. If this is part of a whole list of things, you could consider doing it that way too.

Reply:
Swapped 'em out!